src/lib/sw_engine/tvgSwFill.cpp

```diff
@@ -144,13 +144,13 @@ bool _prepareRadial(SwFill* fill, const RadialGradient* radial, const Matrix* tr
         fill->radial.cx = tx;
         fill->radial.cy = ty;
 
-        auto sx = sqrt(pow(transform->e11, 2) + pow(transform->e21, 2));
```

Comment:
this case, probably it needs powf().
double pow(double x, double y);
float powf(float x, float y);